### PR TITLE
Added sign up ui component

### PR DESCRIPTION
**Pull request by mwangimuringi**
Styled it also.

### CODE REVIEW

File: client/src/index.css
Comment (muringitech) [inline client/src/index.css:1]: ok

File: client/src/pages/SignUp.jsx
Comment (muringitech) [inline client/src/pages/SignUp.jsx:1]: nice

**Review by muringitech — APPROVED** [review]
awesome.

File: client/src/index.css
Comment (mwangimuringi) [reply client/src/index.css:1]: ok

File: client/src/pages/SignUp.jsx
Comment (mwangimuringi) [reply client/src/pages/SignUp.jsx:1]: good

**Pull request merged**